libraries/ts/src/margin/marginClient.ts

```diff
@@ -74,7 +74,7 @@ export class MarginClient {
 
   static getConfig(cluster: MarginCluster): MarginConfig {
     if (typeof cluster === "string") {
-      return JET_CONFIG[cluster] as MarginConfig
+      return JET_CONFIG[cluster] as unknown as MarginConfig
```

Comment:
It needs to compile sir. Your interfaces don't match the json the initializer created. You can get it to compile here, but if you use it in a frontend it'll throw errors there anyway. Don't turn off the typechecker
```suggestion
      return JET_CONFIG[cluster] as MarginConfig
```